src/main/java/org/commonjava/storage/pathmapped/datastax/CassandraPathDB.java

```diff
@@ -253,14 +253,14 @@ public boolean delete( String fileSystem, String path )
     private ReverseMap deleteFromReverseMap( String fileId, String path )
     {
         logger.debug( "Delete from reverseMap, fileId: {}, path: {}", fileId, path );
-        session.execute( "UPDATE " + keyspace + ".reversemap SET paths -= {'" + path + "'} WHERE fileid=?;", fileId );
+        session.execute( "UPDATE " + keyspace + ".reversemap SET paths = paths - {'" + path + "'} WHERE fileid=?;", fileId );
```

Comment:
Just double confirm: is this a typo or intentional? If this is intentional please ignore this comment.

Reply:
Yeah, this fix is intentional. The += and -= only works on the higher CQL version. When I tested with a lower version Cassandra, it raise Syntax error.